chall03/srouhe.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+	"strconv"
+	"net/http"
+	"io/ioutil"
+)
+
+
+func errCheck(err error) {
+	if err != nil {
+        log.Fatalf("Fatal error: %v", err)
+    }
+}
```

Comment:
The idea is good, but unfortunately, this is not following the Golang best practice. It can sound weird because it forces you to go against the DRY concept, but it's better to keep your error handling right under where you use it.

Some errors could just raise a warning and not os.Exit(1) (what log.Fatalf is doing) so this function wouldn't work.